tools/all_questions.xml

```diff
@@ -2099,8 +2128,8 @@ Ebenso wie bei den Formaten gilt hier: je standardisierter, offener und etablier
 		<questionset dc:uri="https://rdmorganiser.github.io/terms/questions/ua_ruhr/data-usage/quality-assurance-integration"/>
 		<is_collection>False</is_collection>
 		<order>0</order>
-		<help lang="en"/>
-		<help lang="de"/>
+    <help lang="en">For example, how is the origin and quality of the data documented?</help>
+    <help lang="de">Wie wird z. B. Herkunft und Qualität der Daten dokumentiert?</help>
```

Comment:
"z.B."  is written without space in other strings. Not sure if this is relevant.

Reply:
merged in from upstream. We should file an issue there.